packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -669,16 +674,16 @@ export function useClaimCctp(tx: MergedTransaction) {
 }
 
 export function getL1ChainIdFromSourceChain(tx: MergedTransaction) {
-  if (!tx.cctpData?.sourceChainId) {
-    return ChainId.Ethereum
+  switch (tx.parentChainId) {
```

Comment:
Does this keep the exact same behaviour? It's weird that we're checking for `parentChainId` but there's case for Arbitrum networks. This should only happen with Orbit

Reply:
You're right it's redundant for cctp.

I changed it so that we use `tx.parentChainId` instead. Before CCTP didn't have access to this but now we can use that instead.